graph-db/indexer/handler.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports.index = async event => {
+  console.log(`Ryan, Sarah and Mark will see ingest event: [${JSON.stringify(event, null, 2)}].`)
+  return {
+    statusCode: 200,
+    body: JSON.stringify(
+      {
+        message: 'Processed ingest event successfully!',
+        input: event,
```

Comment:
What is the purpose of returning the event? If you are returning more than a 200 should that jut be the request body?

Reply:
I'm going to be modifying this, so it shouldn't actually matter what is here

Reply:
See my comment: This function will be completely replaced with the actual indexing code that is coming when we work on CMR-7307. So ignore what's happening inside here.